p2p/trust/trustmetric.go

```diff
@@ -381,9 +384,7 @@ func NewMetricWithConfig(tmc TrustMetricConfig) *TrustMetric {
 	tm.historyMaxSize = intervalToHistoryOffset(tm.maxIntervals) + 1
 	// This metric has a perfect history so far
 	tm.historyValue = 1.0
-	// Setup the channels
-	tm.update = make(chan *updateBadGood, defaultUpdateChanCapacity)
```

Comment:
we can get rid of the corresponding constants defined at the top of the file

Reply:
will do